django_project/settings.py

```diff
@@ -41,8 +41,16 @@
     'customers',
     'stays',
     'search',
+    'rest_framework',
```

Comment:
In order for this to be consistent please move this dependency under `django_extensions`